client/src/components/sourceMapSlice.ts

```diff
@@ -6,10 +6,12 @@ export enum ViewType {cluster, trace}
 
 const initialState: View = {
   type: ViewType.cluster,
+  data: 'placeholder'
```

Comment:
can this just initialize as undefined?

Reply:
It looks like, on line 14, that it's an optional parameter so probably no need for 'placeholder?

Reply:
initialized as undefined.